.travis.yml

```diff
@@ -26,11 +27,8 @@ matrix:
       env: SYMFONY_VERSION='~4.4.0'
     - php: '7.2'
       env: SYMFONY_VERSION='~5.0.0'
-  allow_failures:
-    - php: nightly
 
 before_install:
-  - set -eo pipefail
```

Comment:
without it, it means you can have an error from `covers-validator` for example but still carrying on. I'm not sure that's a good idea

Reply:
that's not true. `covers-validator` is in the `install` section, so a failure there already mark the build as being in the error state and stops it.
But having this here actually breaks Travis: a failing command in the `script` section was not properly marking the build as being in the `failed` state (and if you were trying to use `after_failure`, it would not work either).

On a side node, I think that `covers-validator` should be a `script` step rather than an `install` step (considering it one of the test commands triggering the `failed` state but letting other tests run). To me, it is perfectly fine to run your testsuite even if a `@covers` annotation is missing so that you get the full feedback from the CI.

Reply:
>To me, it is perfectly fine to run your testsuite even if a @covers annotation is missing so that you get the full feedback from the CI.

It's fine when you are in the logic of letting the CI give you all the mistakes, but it's not really the logic I'm going for.

> But having this here actually breaks Travis: a failing command in the script section was not properly marking the build as being in the failed state (and if you were trying to use after_failure, it would not work either).

Interesting to know... So what is the proper way to fail fast on a multi-step scripts?

>I think that covers-validator should be a script step rather than an install step 

Agreed, not sure why it's there

Reply:
well, the `scripts` section is precisely meant to run all the various scripts, so that you get all your test failures in the CI output.
If you want it to stop the build, keep it in `install`

Reply:
https://travis-ci.org/nelmio/alice/builds/658807997 proves it keeps breaking the build btw (and it now properly renders the Travis message because Travis ends the build cleanly rather than `pipefail` making the build script crash)

Reply:
Let's remove it then